src/types/ortb2.d.ts

```diff
@@ -0,0 +1,41 @@
+export namespace Ortb2 {
```

Comment:
I've allowed myself to slightly improve the types for ortb 2 object in bidder request (according to the [docs](https://docs.prebid.org/dev-docs/bidder-adaptor.html#note-on-ortb-adapters-1:~:text=across%20ad%20units.-,ortb2,-is%20the%20global)). If this change goes against your practices, I could send it via an additional PR or just discard. You are welcome to let me know, thanks